src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsService.java

```diff
@@ -48,6 +48,9 @@ public class JenkinsService extends AbstractContinuousIntegrationService {
 
     private static final Logger log = LoggerFactory.getLogger(JenkinsService.class);
 
+    @Value("${artemis.continuous-integration.url}")
+    protected URL serverUrl;
+
```

Comment:
Why do we need this here?

Reply:
I moved it out of the AbstractContinuousIntegrationService because it will not apply to the `LocalCIService`. The test server, that the LocalVC/LocalCI setup runs on, does not even have this environment variable set.